package.json

```diff
@@ -61,7 +61,7 @@
     "vue-template-compiler": "^2.6.11"
   },
   "engines": {
-    "node": ">= 8.9.0",
+    "node": ">= 12.0",
```

Comment:
Could omitting the patch version here (instead of `">= 12.0.0"`) lead to problems?

Reply:
See https://docs.npmjs.com/files/package.json#engines 
I currently can't construct a problem that could arise with this version scheme. Do you have a potential problem in mind?

Reply:
I wasn’t sure if this is a simple string comparison or if major, minor and patch are compared individually. Since the latter seems to be the case, it should be fine.